core/src/com/unciv/ui/utils/ImageGetter.kt

```diff
@@ -257,7 +257,7 @@ object ImageGetter {
     fun getTriangle() = getImage("OtherIcons/Triangle")
 
     fun getBackground(color: Color): Drawable {
-        val drawable = getDrawable("OtherIcons/TableBackground")
+        val drawable = getDrawable("")
```

Comment:
Was this intentional?

Reply:
"OtherIcons/TableBackground" is the same as "OtherIcons/whiteDot"

Reply:
"OtherIcons/TableBackground" is the same as "OtherIcons/whiteDot"